### ISSUE TITLE
Only create Stripe Invoices if the currency is USD

### ISSUE THREAD

**Issue by otech47** [setlife-network/trinary#452]
Currently when creating a Payment we check if the client has an `external_uuid` before trying to create a linked Stripe Invoice

We should also add another check on the `currency` as well

We do not care to link Trinary and Stripe unless the currency is USD because Stripe does not support BTC/SATS

We could consider supporting EUR or MEX but we would need to lock the currency field on the Client because Stripe will not allow different currencies to be used after an invoice has been created. If we decide to do this it will be a separate issue

**Comment by sofiarm21** [comment]
In an effort of scoping this issue we will do the following:
1. This issue will only contain the implementation of this `We do not care to link Trinary and Stripe unless the currency is USD`
2. I'll proceed and create a new issue for this `lock the currency field on the Client because Stripe will not allow different currencies to be used after an invoice has been created` after giving it a further thought

**Comment by otech47** [comment]
## From Stripe to Trinary

- When creating Invoices in Stripe, the connected Client in Trinary will have its currency changed to USD (the currency of the invoice) and then will be locked from updating
- Do not create Trinary Payments if there are already payments in a different currency (should we "disconnect" the Client from the Customer in this case and delete the `external_uuid`)

## From Trinary to Stripe

- When creating a Trinary Payment, it should only create a Stripe invoice if the currency is supported (USD, EUR, MXN) only after updating the currency of the Stripe Customer which will then be locked after the invoice is created
